infrastructure/tooling/src/smoketest/checks/indexTask.js

```diff
@@ -0,0 +1,60 @@
+const taskcluster = require('taskcluster-client');
+
+exports.scopeExpression = {
+  AllOf: [
+    'queue:create-task:highest:built-in/succeed',
+    'queue:scheduler-id:-',
+  ],
+};
+
+exports.tasks = [];
+exports.tasks.push({
+  title: 'Create an indexed task and find it in the index (--target index)',
+  requires: [
+    'ping-queue',
+    'ping-index',
+  ],
+  provides: [
+    'target-index',
+  ],
+  run: async (requirements, utils) => {
+    let queue = new taskcluster.Queue(taskcluster.fromEnvVars());
+    let randomId = taskcluster.slugid();
+    const taskIndex='project.taskcluster.smoketest.' + randomId;
+    let task = {
+      provisionerId: 'built-in',
+      workerType: 'succeed',
+      created: (new Date()).toJSON(),
+      deadline: taskcluster.fromNowJSON('2 minutes'),
+      expires: taskcluster.fromNowJSON('60 minutes'),
+      metadata: {
+        name: "Smoketest indexTask-find",
+        description: "built-in/succeed task created during smoketest",
+        owner: "smoketest@taskcluster.net",
+        source: "https://taskcluster.net",
+      },
+      payload: {},
+      routes: [`index.${taskIndex}`],
+    };
+    utils.status({message: 'indexTask-find taskId: ' + randomId});
+    await queue.createTask(randomId, task);
+    let index = new taskcluster.Index(taskcluster.fromEnvVars());
+    await new Promise(resolve => setTimeout(resolve, 5000));
```

Comment:
I commented this out, and the test failed.  That means that something in this test is dependent on timing, and that often leads to intermittent test failures.  Can you remove this and fix the issue causing the test to fail?